sdk/tools/widl/proxy.c

```diff
@@ -739,7 +734,7 @@ static void write_proxy(type_t *iface, unsigned int *proc_offset)
       print_proxy( "static const PRPC_STUB_FUNCTION %s_table[] =\n", iface->name);
       print_proxy( "{\n");
       indent++;
-#ifdef __REACTOS__ /* r57019 / c3be8a3 */
+#ifdef __REACTOS__
```

Comment:
```suggestion
#ifdef __REACTOS__ /* r57019 / c3be8a3 */
```
Isn't the comment still accurate? (pointing to ROS commit where this has been added)